src/sidebar/test/media-embedder-test.js

```diff
@@ -90,6 +90,24 @@ describe('media-embedder', function () {
     });
   });
 
+  it('replaces internet archive links with iframes', function () {
+    var urls = [
+      'https://archive.org/details/WHDH_20151121_043400_The_Tonight_Show_Starring_Jimmy_Fallon/start/360/end/420.3',
+      'https://archive.org/embed/WHDH_20151121_043400_The_Tonight_Show_Starring_Jimmy_Fallon?start=360&end=420.3',
+    ];
+    urls.forEach(function (url) {
+      var element = domElement('<a href="' + url + '">' + url + '</a>');
+
+      mediaEmbedder.replaceLinksWithEmbeds(element);
+
+      assert.equal(element.childElementCount, 1);
+      assert.equal(element.children[0].tagName, 'IFRAME');
+      assert.equal(
+        element.children[0].src, 'https://archive.org/embed/WHDH_20151121_043400_The_Tonight_Show_Starring_Jimmy_Fallon?start=360&end=420.3');
+    });
+  });
+
```

Comment:
We need additional tests.
There are 4 different code paths in this function. We need to test all of them.
Also, different tests that cover the regex would be useful.

Reply:
I've explained in comments that the test covers both flavors of URL. What would you add to that?